libs/blocks/columns/columns.js

```diff
@@ -1,19 +1,74 @@
+const hasMeaningfulChildren = (col) => Array.from(col.childNodes).filter(
```

Comment:
The `has` makes me think this returns a Boolean, when in fact it returns an Array, so maybe `getMeaningfulChildren` would be more accurate?